example_code/shmem_scan_example.c

```diff
@@ -0,0 +1,12 @@
+#include <shmem.h>
+
+int collect_at(shmem_team_t team, void *dest, const void *source, size_t nbytes, int who) {
+  static size_t sym_nbytes;
+  sym_nbytes = nbytes;
+  shmem_team_sync(team);
```

Comment:
Can we get rid of this sync if the src and dest buffers are different and dest is statically initialized?

If not, we may need to address this case in the section matter above.